core-java-modules/core-java-lang-oop-constructors/src/main/java/com/baeldung/initializerblock/staticblock/StaticBlockExample.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.initializerblock.staticblock;
+
+public class StaticBlockExample {
+
+    static {
+        System.out.println("static block 1");
+    }
+    static {
```

Comment:
Same thing here. The static blocks needs to have an empty line in between.

Reply:
Updated.